state/futures/actions.ts

```diff
@@ -366,6 +366,47 @@ export const fetchMarginTransfers = createAsyncThunk<
 	}
 });
 
+export const fetchCombinedMarginTransfers = createAsyncThunk<
+	| {
+			isolatedMarginTransfers: MarginTransfer[];
+			smartMarginTransfers: MarginTransfer[];
+			idleTransfers: MarginTransfer[];
+			context: AccountContext;
+	  }
+	| undefined,
+	void,
+	ThunkConfig
+>('futures/fetchCombinedMarginTransfers', async (_, { getState, extra: { sdk } }) => {
+	const { wallet, futures } = getState();
+	const supportedNetwork = selectFuturesSupportedNetwork(getState());
+	const network = selectNetwork(getState());
+	const cmAccount = selectCrossMarginAccount(getState());
+	if (!wallet.walletAddress || !supportedNetwork) return;
+	try {
+		const isolatedMarginTransfers = await sdk.futures.getIsolatedMarginTransfers();
+		const smartMarginTransfers = cmAccount
+			? await sdk.futures.getIsolatedMarginTransfers(cmAccount)
```

Comment:
Should this be `getCrossMarginTransfers` ?

Reply:
No, we call `isolatedMarginTransfers` specifying the smart margin account address to get transfers from `smart margin account <-> market` and `crossMarginTransfers` to get transfers from `wallet <-> smart margin account`. You can see this in the other action as well.